### ISSUE TITLE
Explore the data - basic EDA

### ISSUE THREAD

**Issue by Sum02dean** [Sum02dean/MLG#16]
We need to understand data, and how we may represent the data in a machine learning appropriate manner.

**Comment by TaoDFang** [comment]
I import bed data to IGV, why all signals display same height ?

**Comment by LiineKasak** [comment]
yeah we ignore the data files right now since otherwise Dean will run out of space in github and would have to pay, I've run into that before haha. and the project updates would get large. but if the file is relatively small and useful for everyone then maybe we can put it under data_small/ ? or we can just share them in slack, maybe in a new channel

**Comment by LiineKasak** [comment]
or, as I did, I added the link to the data into the readme and said where it should be under what name

**Comment by TaoDFang** [comment]
ah. i will create a new folder for this and created a new branch for this issue, but mainly to test new branch workflow
